tests/approaches/test_oracle_approach.py

```diff
@@ -170,6 +170,16 @@
         "pddl_easy_delivery_procedural_test_max_extra_newspapers": 1,
     },
 ]
+EXTRA_ARGS_ORACLE_APPROACH["behavior_tasks"] = [
```

Comment:
i'm confused -- isn't behavior excluded from tests because we didn't install it on the github servers? is this change doing anything?

Reply:
"It fixes so coverage complaints" -Willie

Reply:
Can you be specific?